hub/config/src/main/java/uk/gov/ida/hub/config/data/S3ConfigSource.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.ida.hub.config.data;
+
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.model.GetObjectRequest;
+import com.amazonaws.services.s3.model.S3DataSource;
+import com.amazonaws.services.s3.model.S3Object;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import uk.gov.ida.hub.config.ConfigConfiguration;
+import uk.gov.ida.hub.config.domain.remoteconfig.RemoteConfigCollection;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+
+/**
+ * Configure S3 connection here
+ */
+public class S3ConfigSource{
```

Comment:
```suggestion
public class S3ConfigSource {
```
some kind of linter would be useful :)